wgpu-core/src/command/bundle.rs

```diff
@@ -220,6 +211,30 @@ impl RenderBundleEncoder {
                         .map_pass_err(scope);
                     }
 
+                    // Check for misaligned offsets.
+                    for (offset, info) in offsets
+                        .iter()
+                        .map(|offset| *offset as wgt::BufferAddress)
+                        .zip(bind_group.dynamic_binding_info.iter())
+                    {
+                        let (alignment, limit_name) = match info.binding_type {
```

Comment:
could we avoid repeating this `match` 3 times in the code?

Reply:
Updated in 15cfcc3ab5055ca387ed4c0ce037fb9ffaff1baa, does that look good?